diff --git a/dist/index.mjs b/dist/index.mjs
index 03667c14f63fe3949673c1bb70811fd238b85f6c..ae26642f8052365996e4b263c424525d5f061f42 100644
--- a/dist/index.mjs
+++ b/dist/index.mjs
@@ -331,7 +331,7 @@ function viteLegacyPlugin(options = {}) {
       }
       if (!isLegacyChunk(chunk, opts)) {
         if (options.modernPolyfills && !Array.isArray(options.modernPolyfills) && genModern) {
-          await detectPolyfills(raw, { esmodules: true }, modernPolyfills);
+          await detectPolyfills(raw, options.targets ?? { esmodules: true }, modernPolyfills);
         }
         const ms = new MagicString(raw);
         if (genLegacy && chunk.isEntry) {
